### PR TITLE
Release schnorr-0.7

### PR DESCRIPTION
**Pull request by CPerezz**
(TBH, no idea about what 8e8be9c is doing here... maybe @vlopes11 branched out from a stale `main` branch or something..)

Resolves: #43

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,7 +7,26 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Update `dusk-plonk` from `0.6` to `0.8` [#41](https://github.com/dusk-network/schnorr/issues/41)
+- Update `dusk-poseidon` from `0.18` to `0.21.0-rc` [#41](https://github.com/dusk-network/schnorr/issues/41)
+- Update `dusk-pki` from `0.6` to `0.7` [#41](https://github.com/dusk-network/schnorr/issues/41)
+- Change crate name from `schnorr` to `dusk-schnorr` [#41](https://github.com/dusk-network/schnorr/issues/41)
+### Removed
+- Remove `dusk-bls12_381` and import it from `dusk-plonk` [#41](https://github.com/dusk-network/schnorr/issues/41)
+- Remove `dusk-jubjub` and import it from `dusk-plonk` [#41](https://github.com/dusk-network/schnorr/issues/41)
+- Remove one hashing level for `message` in signature processing [#55](https://github.com/dusk-network/schnorr/issues/55)
+
+### Removed
```
Comment (ZER0) [inline CHANGELOG.md:24]: You have two removed section instead of one.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,7 +7,26 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Remove `dusk-jubjub` from dependencies [#50](https://github.com/dusk-network/schnorr/issues/50)
+- Remove `dusk-bls12_381` from dependencies [#50](https://github.com/dusk-network/schnorr/issues/50)
+- Remove `anyhow` from dependencies [#50](https://github.com/dusk-network/schnorr/issues/50)
+- Change default crate featureset to be `alloc`. [#50](https://github.com/dusk-network/schnorr/issues/50)
```
Comment (ZER0) [inline CHANGELOG.md:28]: This belongs to Changed section.

File: src/key_variants/double_key.rs [hunk 3]
```diff
@@ -6,21 +6,19 @@
...
-#[cfg(feature = "canon")]
-use canonical::Canon;
 #[cfg(feature = "canon")]
 use canonical_derive::Canon;
-#[allow(unused_imports)]
-use dusk_bls12_381::BlsScalar;
 use dusk_bytes::{DeserializableSlice, Error as BytesError, Serializable};
-use dusk_jubjub::{JubJubScalar, GENERATOR_EXTENDED, GENERATOR_NUMS_EXTENDED};
 use dusk_pki::{PublicKey, SecretKey};
+use dusk_plonk::bls12_381::BlsScalar;
```
Comment (ZER0) [inline src/key_variants/double_key.rs:13]: I thought we decided to do not use `BlsScalar` (or JubJub) from plonk anymore.

File: src/key_variants/double_key.rs [hunk 4]
```diff
@@ -6,21 +6,19 @@
...
+use dusk_plonk::jubjub::{
```
Comment (ZER0) [inline src/key_variants/double_key.rs:14]: Same here.

File: src/key_variants/single_key.rs [hunk 5]
```diff
@@ -4,26 +4,23 @@
...
-#[cfg(feature = "canon")]
-use canonical::Canon;
 #[cfg(feature = "canon")]
 use canonical_derive::Canon;
-use dusk_bls12_381::BlsScalar;
 use dusk_bytes::{DeserializableSlice, Error as BytesError, Serializable};
-use dusk_jubjub::{
+use dusk_pki::{PublicKey, SecretKey};
+use dusk_plonk::bls12_381::BlsScalar;
+use dusk_plonk::jubjub::{
```
Comment (ZER0) [inline src/key_variants/single_key.rs:12]: Same here.

File: tests/double_key_tests.rs [hunk 6]
```diff
@@ -4,9 +4,9 @@
 //
 // Copyright (c) DUSK NETWORK. All rights reserved.
 
-use dusk_bls12_381::BlsScalar;
 use dusk_pki::SecretKey;
-use schnorr::{Proof, PublicKeyPair};
+use dusk_plonk::bls12_381::BlsScalar;
```
Comment (ZER0) [inline tests/double_key_tests.rs:8]: Same here and in the rest of the file. But before proceed let's discuss in the chat, I think we all agreed; but let's double check.

**Review by ZER0 — CHANGES_REQUESTED** [review]
LGTM but there are few nits.

**Review by vlopes11 — APPROVED** [review]
LGTM

Consider rebasing and reducing the number of commits in this PR. We have a disproportional number of commits compared to the changeset

**Review by ZER0 — APPROVED** [review]
LGTM

**Pull request merged**